Tests/Functional/Bundle/TestBundle/Controller/ParamFetcherController.php

```diff
@@ -23,13 +23,13 @@
 class ParamFetcherController extends FOSRestController
 {
     /**
-     * @RequestParam(name="raw", requirements=@IdenticalTo({"foo"="raw", "bar"="foo"}), default="invalid")
-     * @RequestParam(name="map", map=true, requirements=@IdenticalTo({"foo"="map", "foobar"="foo"}), default="%invalid2% %%")
-     * @RequestParam(name="bar", map=true, requirements="%bar% foo", strict=true)
+     * @RequestParam(name="raw", requirements=@IdenticalTo({"foo"="raw", "bar"="foo"}), default="invalid", strict=false)
+     * @RequestParam(name="map", map=true, requirements=@IdenticalTo({"foo"="map", "foobar"="foo"}), default="%invalid2% %%", strict=false)
+     * @RequestParam(name="bar", nullable=true, requirements="%bar%\ foo")
```

Comment:
Btw this is really not expected. It is due to the `x` option and is not convenient imo, unfortunately it's gonna be hard to change...